database/seeds/05-tags_courses.js

```diff
@@ -15,6 +15,18 @@ exports.seed = function(knex, Promise)
                 tag_id: 3,
                 course_id: 1,
```

Comment:
I didn't know you could have a primary key be two ids,  I learned something new with this.